relays/finality-relay/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "finality-relay"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+description = "Finality proofs relay"
```

Comment:
```suggestion
description = "Relay finality proofs from a source Substrate chain to a target Substrate chain."
```

Reply:
This is clearly wrong - this crate knows nothing about Substrate

Reply:
Yeah, I guess it doesn't know anything about it - but as humans we know, and it could be helpful if we publish these crates at some point.

Reply:
As a human, I'm saying that this part: "from a source Substrate chain to a target Substrate chain" is clearly wrong.